controller/contour/namespace.go

```diff
@@ -35,19 +35,16 @@ const (
 // ensureNamespace ensures the namespace for the provided name exists.
 func (r *Reconciler) ensureNamespace(ctx context.Context, name string) error {
 	ns := &corev1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: name}}
-	err := r.Get(ctx, types.NamespacedName{Name: ns.Name}, ns)
-	switch {
-	case err == nil:
-		r.Log.Info("namespace exists; skipped adding", "name", ns.Name)
-		return nil
-	case errors.IsNotFound(err):
+	if err := r.Get(ctx, types.NamespacedName{Name: ns.Name}, ns); err != nil {
```

Comment:
@danehans Did you intend to include these changes?

Reply:
@jpeach I did not, it must have accidentally been included when I updated the PR for the DCO. Thanks for catching the mistake. I've updated the PR to include only the .gitignore.